addons/smtp-mail.sh

```diff
@@ -213,9 +213,9 @@ sudo chmod 0644 $VMLOGS/mail_msmtp.log
 
 # Create aliases
 cat << ALIASES_CONF > /etc/aliases
-root: $RECIPIENT
-default:$RECIPIENT
-cron: $RECIPIENT
+root: $MAIL_USERNAME
+default:$MAIL_USERNAME
+cron: $MAIL_USERNAME
```

Comment:
the aliases file is for sending mails and not the recipient, afaik.

Reply:
aliases [file]
              Replace local recipients with addresses in the aliases file.  The aliases file is a
              plain  text  file  containing mappings between a local address and a list of domain
              addresses.  A local address is defined as one without an `@' character and a domain
              address is one with an `@' character.  The mappings are of the form:
                  local: someone@example.com, person@domain.example
              Multiple  domain  addresses are separated with commas.  Comments start with `#' and
              continue to the end of the line.
              The local address default has special significance and  is  matched  if  the  local
              address  is  not found in the aliases file.  If no default alias is found, then the
              local address is left as is.
              An empty argument  to  the  aliases  command  disables  the  replacement  of  local
              addresses.  This is the default.

http://manpages.ubuntu.com/manpages/precise/en/man1/msmtp.1.html

Reply:
In other words, please revert.

Reply:
Please read this:
https://mytechiethoughts.com/linux/simple-way-to-get-email-notifications-from-a-linux-server/#the-alias-file

Reply:
<!-- Error while uploading Screenshot_20201003-122200_Brave.png -->

Reply:
![photo5460666311588818772](https://user-images.githubusercontent.com/42591237/94989230-7c511a80-0573-11eb-84b5-232500beb0ae.jpg)

Reply:
if I understand it correctly, we don't need the aliases file at all

Reply:
sry, I was wrong